### PR TITLE
Command add support for external endpoints

### PR DESCRIPTION
**Pull request by kaweue**

### CODE REVIEW

File: pkg/controller/command/command_config_maps.go [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 
+	"github.com/google/uuid"
 	corev1 "k8s.io/api/core/v1"
 
 	contrail "github.com/Juniper/contrail-operator/pkg/apis/contrail/v1alpha1"
 	"github.com/Juniper/contrail-operator/pkg/certificates"
 	"github.com/Juniper/contrail-operator/pkg/k8s"
 )
 
+var NameSpaceCommand = uuid.Must(uuid.Parse("362ad2d9-9460-4f65-a702-2cd99fbb0f64"))
```
Comment (katrybacka) [inline pkg/controller/command/command_config_maps.go:15]: i don't understand the name of this variable, what did you mean?

File: pkg/controller/command/command_config_maps.go [hunk 2]
```diff
@@ -49,16 +53,31 @@ func (c *configMaps) ensureCommandConfigExist(postgresAddress, ConfigEndpoint st
...
+		{Name: "nodejs", PrivateURL: webUIURL, PublicURL: webUIURL},
+		{Name: "telemetry", PrivateURL: telemetryURL, PublicURL: telemetryURL},
+		{Name: "config", PrivateURL: configAPIURL, PublicURL: configAPIURL},
+		{Name: "keystone", PrivateURL: keystoneURL, PublicURL: keystoneURL},
+		{Name: "swift", PrivateURL: swiftProxyURL, PublicURL: swiftProxyURL},
+	}
+	ces = append(ces, c.ccSpec.ServiceConfiguration.Endpoints...)
+	bes := []bootstrapEndpoint{}
+	for _, e := range ces {
+		bes = append(bes, *newBootstrapEndpoint(e))
```
Comment (katrybacka) [inline pkg/controller/command/command_config_maps.go:72]: do we need to return an pointer from newBootstrapEndpoint method if you are just using objects and then you append the object to the slice

**Review by katrybacka — CHANGES_REQUESTED** [review]

**Comment by lrebeja** [comment]
Maybe we can allow user to configure this port as well?

**Comment by kaweue** [comment]
@kaweue Which port?

File: pkg/controller/command/command_config_maps.go (hunk 1 above)
Comment (kaweue) [reply pkg/controller/command/command_config_maps.go:18]: Added a comment

File: pkg/controller/command/command_config_maps.go (hunk 2 above)
Comment (kaweue) [reply pkg/controller/command/command_config_maps.go]: changed

**Review by kabart — APPROVED** [review]

**Review by katrybacka — APPROVED** [review]

**Pull request merged**